verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -177,6 +177,20 @@ bool aws_cryptosdk_hdr_members_are_bounded(
     return true; /* If hdr is NULL, true by default */
 }
 
+struct aws_cryptosdk_hdr *hdr_setup(
+    const size_t max_table_size, const size_t max_edk_item_size, const size_t max_item_size) {
```

Comment:
`max_item_size` is not used anywhere.

Reply:
Thanks! copy-paste error which is fixed now!